### ISSUE TITLE
Some admin pages have HTML parsing errors

### ISSUE THREAD

**Issue by mcgratay** [pressbooks/pressbooks#1650]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with one of our built-in root or book themes?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

In content implemented using markup languages, elements must have complete start and end tags, be nested according to their specifications, not contain duplicate attributes, and have unique IDs, except where the specifications allow these features. These types of errors can interfere with the operation of assistive technologies.

The Book Information page contains 247 parsing errors and warning, including: 
- Duplicate ID values
- Malformed elements
- Misused attributes

In general, ensure valid code by checking pages with the W3C Nu Validator (https://validator.w3.org/nu/). The raw results show all of the source code validity issues, but to Pass this criterion only a subset of these issues must be fixed. This subset can be filtered using a special “bookmarklet” found at: http://www.paciellogroup.com/blog/2012/02/wcag-2-0-parsing-error-bookmarklet/

**Comment by SteelWagstaff** [comment]
The ARIA-level error should be separated into its own issue--this is something that occurs in the Tiny MCE editor we use, and can't be fixed in the same way that malformed HTML can be.

**Comment by mcgratay** [comment]
One remaining error: 
![Screen Shot 2019-04-17 at 2 49 20 PM](https://user-images.githubusercontent.com/39064028/56313266-3bb3d600-6120-11e9-81f8-450b1c2f75b3.png)
![Screen Shot 2019-04-17 at 2 49 33 PM](https://user-images.githubusercontent.com/39064028/56313270-3f475d00-6120-11e9-9280-884e7e645804.png)

**Comment by mcgratay** [comment]
Resolved 👍
